tests/test__mongomock.py

```diff
@@ -1680,7 +1681,11 @@ def test__aggregate17(self):
         pipeline = [
             {'$project': {'_id': 0, 'created': {'$subtract': [{'$min': ['$a', '$b']}, '$count']}}}
         ]
-        self.cmp.compare.aggregate(pipeline)
+        # TODO(Metrodatateam): use skipif if version is accessible from decorator
```

Comment:
?

Reply:
some aggregate was not supported until certain version, so I think we may better just skip some tests if it's not supported

Reply:
Ok, so can we please replace this TODO with an open issue?